actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -59,13 +59,11 @@ def initialize(routes)
         @routes = routes
       end
 
-      def format(formatter, filter = nil)
-        routes_to_display = filter_routes(filter)
-
+      def format(formatter, **options)
+        routes_to_display = filter_routes(options)
         routes = collect_routes(routes_to_display)
-
         if routes.none?
-          formatter.no_routes(collect_routes(@routes), filter)
+          formatter.no_routes(collect_routes(@routes), options[:pattern])
```

Comment:
Let's look at making the no_routes message better instead. I don't think we should output the controller name because then we'd have to worry about escaping as @matthewd said in the original pull request.